pyrate/core/config.py

```diff
@@ -502,6 +515,72 @@ def coherence_paths_for(path: str, params: dict, tif=False) -> str:
     return coh_file_paths[0]
 
 
+def baseline_paths_for(path: str, params: dict) -> str:
+    """
+    Returns path to baseline file for given interferogram. Pattern matches
+    based on epoch in filename.
+
+    Example:
+        '20151025-20160501_base.par'
+        Date pair is the epoch.
+
+    Args:
+        path: Path to intergerogram to find baseline file for.
+        params: Parameter dictionary.
+        tif: Find converted tif if True (_cc.tif), else find .cc file.
+
+    Returns:
+        Path to coherence file.
+    """
+
+    _, filename = split(path)
+    epoch = re.search(sixteen_digits_pattern, filename).group(0)
+    bas_file_paths = [f.unwrapped_path for f in params[BASELINE_FILE_PATHS] if epoch in f.unwrapped_path]
+
+    if len(bas_file_paths) > 2:
+        raise ConfigException(f"'{BAS_FILE_DIR}': found more than one baseline "
+                      f"file for '{path}'. There must be only one "
+                      f"baseline file per interferogram. Found {bas_file_paths}.")
+    return bas_file_paths[0]
+
+
+def mlooked_path(path, xlooks, ylooks, crop_opt):
+    """
+    Adds suffix to ifg path, for creating a new path for multilooked files.
+
+    :param str path: original interferogram path
+    :param int xlooks: number of X looks applied
+    :param int ylooks: number of Y looks applied
+    :param int crop_opt: crop option applied
+
+    :return: multilooked file name
+    :rtype: str
+    """
+    base, ext = splitext(path)
+    return "{base}_{xlooks}lksx_{ylooks}lksy_{crop_opt}cr{ext}".format(base=base, xlooks=xlooks, ylooks=ylooks, crop_opt=crop_opt, ext=ext)
+
+
+def get_dest_paths(base_paths, crop, params, looks):
+    """
+    Determines the full path names for the destination multilooked files
+
+    :param list base_paths: original interferogram paths
+    :param int crop: Crop option applied
+    :param dict params: Parameters dictionary
+    :param int looks: number of range looks applied
+
+    :return: full path names for destination files
+    :rtype: list
+    """
+
+    dest_mlooked_ifgs = [mlooked_path(os.path.basename(q).split('.')[0] + '_'
+                                      + os.path.basename(q).split('.')[1] +
+                                      '.tif', looks=looks, crop_opt=crop)
+                         for q in base_paths]
+
+    return [os.path.join(params[OUT_DIR], p) for p in dest_mlooked_ifgs]
+
+
```

Comment:
these two functions were deprecated in release 0.5.0. They don't exist in the develop branch

Reply:
Thanks for clarifying. That was one of the merge conflicts which I wasn't sure about. The new functionality doesn't need these functions either. They can hence be deleted.

Reply:
see https://github.com/GeoscienceAustralia/PyRate/pull/302/commits/fa2a65850f4aec5bfc26712c9339f19b6edc51ee